scripts/suite_objects.py

```diff
@@ -1535,6 +1535,16 @@ def add_var_transform(self, var, compat_obj, vert_dim):
 
         # If needed, modify vertical dimension for vertical orientation flipping
         _, vdim    = find_vertical_dimension(var.get_dimensions())
+        if vdim >= 0:
+           vdim_name  = vert_dim.split(':')[-1]
+           group_vvar = self.__group.call_list.find_variable(vdim_name)
+           vname      = group_vvar.get_prop_value('local_name')
+           lindices[vdim] = '1:'+vname
+           rindices[vdim] = '1:'+vname
+           if compat_obj.has_vert_transforms:
+               rindices[vdim] = vname+':1:-1'
+           # end if
+        # end if
```

Comment:
Do vertical dimensions have to begin at `1`? Should we use variable starting point if provided?
```suggestion
            vdims  = vert_dim.split(':')
            vdim_name  = vdims[-1]
            group_vvar = self.__group.call_list.find_variable(vdim_name)
            if group_vvar is None:
                raise CCPPError(f"add_var_transform: Cannot find dimension variable, {vdim_name}")
            # end if
            vname = group_vvar.get_prop_value('local_name')
            if len(vdims) == 2:
                sdim_name = vdims[0]
                group_vvar = self.find_variable(sdim_name)
                if group_vvar is None:
                    raise CCPPError(f"add_var_transform: Cannot find dimension variable, {sdim_name}")
                # end if
                sname = group_vvar.get_prop_value('local_name')
            else:
                sname = '1'
            # end if
            lindices[vdim] = sname+':'+vname
            if compat_obj.has_vert_transforms:
                rindices[vdim] = vname+':'+sname+':-1'
            else:
                rindices[vdim] = sname+':'+vname
            # end if
        # end if
```

Reply:
smart! changed.